README.md

```diff
@@ -17,114 +17,10 @@ git clone git@github.com:neuromation/ml-recipe-midi-generator.git
 cd ml-recipe-midi-generator
 ```
 
+It will ask you to log in on Neuro Platform when the command is run for the first time. A Github account can be used for this purpose.
+
 This repository already contains pre-trained models, so that you can run Jupyter Notebook with inference code and play with these models. To do so, just copy the following command:
 
 ```
-make setup && make upload && make jupyter
+make setup && make upload-all && make jupyter
 ```
-
```

Comment:
I know this is more project-specific, but this info mostly duplicates `HELP.md`...

Reply:
I still think that the default `README.md`, which is called `HELP.md` now,  is unclear. My explanation without mentioning unused functionalities is better for understanding when we are talking about new users.